src/main/java/com/jalasoft/webservice/controller/ImageController.java

```diff
@@ -46,6 +48,7 @@ public Response Convert (@RequestParam("file") MultipartFile file,
                              @RequestParam (value = "dpi", defaultValue = "") int dpi,
                              @RequestParam(value = "ext", defaultValue = "") String ext) throws IOException,
                              NoSuchAlgorithmException, TikaException, SAXException {
+            logger.info("Starting Image Controller - Method: " + new Object() {}.getClass().getEnclosingMethod().getName());
```

Comment:
review this line do not overload the 100 character

Reply:
Done